environment_py2_linux.yml

```diff
@@ -25,5 +25,6 @@ dependencies:
   - xarray>=0.10.8
   - dask
   - cftime
+  - psutil
```

Comment:
Since the PR has been open for a while, should we check if this dependency is still necessary?

Reply:
We need this package to check on the amount of free virtual memory: https://github.com/OceanParcels/parcels/blob/ab2976a63584ca3f8590282a862abd674fd3ddfa/parcels/particlefile.py#L324 

Unless anyone knows a python-core method to do this, I think we'll have to keep this dependency